pkg/api/api.go

```diff
@@ -41,6 +41,7 @@ var apiMap = map[string]string{
 	// Environment API not Config API
 	"synthetic-monitor":  "/api/v1/synthetic/monitors",
 	"application":        "/api/config/v1/applications/web",
```

Comment:
:crying_cat_face: I guess we missed the opportunity to call this `application-web`.

Reply:
Didn't want to cause a breaking change in an intermediary release. Perhaps a candidate for v2.0.0?

Reply:
:+1: Yeah, for 2.0 we could deprecate `application` and print a warning it its still used - and introduce `application-web` additionally.

Reply:
added application-web endpoint